tools/functional-tester/etcd-tester/tester.go

```diff
@@ -41,80 +41,15 @@ func (tt *tester) runLoop() {
 		tt.status.Failures = append(tt.status.Failures, f.Desc())
 	}
 
-	var (
-		round          int
-		prevCompactRev int64
-	)
-	for {
+	var prevCompactRev int64
+	for round := 0; round < tt.limit; round++ {
```

Comment:
`for round := 0; round < tt.limit; round++ {`

->

`for round := 0; round < tt.limit || tt.limit == -1; round++ {`

So run inifinitely in case we don't pass `limit` flag?

https://github.com/heyitsanthony/etcd/blob/064f50063b6a89fd869e05d7a0d6a7ac9877022e/tools/functional-tester/etcd-tester/main.go#L34